cdap-master/src/main/java/co/cask/cdap/data/tools/UpgraderMain.java

```diff
@@ -338,4 +360,17 @@ private Store getStore() {
     }
     return store;
   }
+
+  /**
+   * gets the {@link FileMetaDataManager} to update log meta
+   *
+   * @return {@link FileMetaDataManager}
+   */
+  private FileMetaDataManager getFileMetaDataManager() {
```

Comment:
Because the LogSaverTableUtil which gets used we want it to use our custom DSFramework.